Dockerfile

```diff
@@ -24,6 +24,9 @@ RUN /output/install-from-bindep \
 # Prepare the /runner folder, seed the folder with demo data
 ADD demo /runner
 
+RUN groupadd -g 555 runner && useradd -u 555 -g 555 runner && \
```

Comment:
Should we consider making this configurable? I am thinking of downstream, and could be possible they may be different.

Reply:
Come to think of it, I don't think we need to specifically set this at all anymore. My original approach was using `podman unshare`, which I promptly noped out of. Let me test with it.